s3/replication/common/tests/system/load_test_async_put.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import asyncio
+from config import Config
+import os
+import sys
+import time
+from object_generator import FixedObjectDataGenerator
+from s3replicationcommon.s3_common import S3RequestState
+from s3replicationcommon.log import setup_logger
+from s3replicationcommon.s3_site import S3Site
+from s3replicationcommon.s3_session import S3Session
+from s3replicationcommon.s3_put_object import S3AsyncPutObject
+
+
+async def main():
+
+    config = Config()
+
+    bucket_name = "sourcebucket"
```

Comment:
Do we want to have this as a config, like we have done for other load transfer tests???

Reply:
Fixed.